### PR TITLE
Add documentation to deploy EDC settings as k8 secrets

### PR DESCRIPTION
**Pull request by DominikPinsel**
Signed-off-by: Dominik Pinsel <dominik.pinsel@daimler.com>

### CODE REVIEW

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]

File: edc-dataplane/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Data Plane
+
+The Eclipse Dataspace Connector consists of a **Control Plan** and a **Data Plane** Application.
+While the **Control Plane** manages Data Transfer, the **Control Plane** is responsible for doing the actual data transfer, so that data is never routes through the control plane itself.
```
[the remark this answers is not in the document]
Comment (DominikPinsel) [reply edc-dataplane/README.md]: thanks for your review :) I really missed that

File: edc-dataplane/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+EDC Github Repository https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues
+
+---
+
+**Please note** that some of these issues might already be fixed on the EDC main branch, but are not part of the specific
+EDC commit the Product-EDC uses.
+
+---
+
+[---]
```
[the remark this answers is not in the document]
Comment (DominikPinsel) [reply edc-dataplane/README.md]: Yea, that's better. I will change it :)

File: edc-tests/src/main/resources/deployment/helm/all-in-one/templates/secret.yaml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  EDC_DATASOURCE_TRANSFERPROCESS_PASSWORD: {{ $plato_psql_password | toString | quote }}
+    # see extension https://github.com/catenax-ng/product-edc/tree/develop/edc-extensions/hashicorp-vault
+  EDC_VAULT_HASHICORP_TOKEN: {{ $plato_vault_token | toString | quote }}
+kind: Secret
+metadata:
+  name: aio-plato-control-secret
+  namespace: {{ .Release.Namespace | default "default" | quote }}
+  labels:
+    {{- include "aio.labels" . | nindent 4 }}
+type: Opaque
```
[the remark this answers is not in the document]
Comment (DominikPinsel) [reply edc-tests/src/main/resources/deployment/helm/all-in-one/templates/secret.yaml:37]: That's perfectly fine :) I've changed it

File: edc-tests/src/main/resources/deployment/helm/all-in-one/templates/secret.yaml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+---
+
+apiVersion: v1
+stringData:
+  EDC_VAULT_HASHICORP_TOKEN: {{ $sokrates_vault_token | toString | quote }}
```
[the remark this answers is not in the document]
Comment (DominikPinsel) [reply edc-tests/src/main/resources/deployment/helm/all-in-one/templates/secret.yaml:83]: ah yea, I forgot this one :)

**Review by denisneuling — APPROVED** [review]

**Pull request merged**